projects/go-lib/src/lib/components/go-off-canvas/go-off-canvas.component.scss

```diff
@@ -25,6 +25,22 @@
   z-index: z-index(off-canvas);
 }
 
+.go-off-canvas.large {
+  width: 75vw;
+  background-color: $theme-light-app-bg;
+  color: $base-dark;
```

Comment:
Should we alphabetize these properties? 😬

Reply:
Yeah lemme pull up my handy alphabet reference...

![image](https://user-images.githubusercontent.com/5261826/90433362-b9338180-e099-11ea-86a0-534b968abfa9.png)